Changelog.md

```diff
@@ -76,6 +76,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - extensions/ext/ray_tracing_pipeline: Pass indirect SBT regions as single item reference (#829)
 - Replaced `c_char` array setters with `CStr` setters (#831)
 - `push_next()` functions now allow unsized `p_next` argument (#855)
+- Removed the redundant `ash::extensions` module hierarchy in favor of defining wrappers directly in the extension modules under `ash::vk` (#894)
```

Comment:
Does it make sense to write this below under `### Removed`?

Reply:
I don't think so. Logically we haven't removed any functionality, we've just changed where it's exported from. Tweaked the wording to make that a bit less weird.

Reply:
You are right, and I thought about the same just after placing this comment and leaving. Starting the sentence with `Removed` immediately makes it feel like that (or at least that portion of the) sentence belongs under `# Removed`. Thanks for clearing up the wording!